conversion.qmd

```diff
@@ -2,3 +2,152 @@
 title: "Conversion to and from R data"
 ---
 
+One of the key goals with extendr, is to provide a framework that allows you to write Rust functions, that interact with R, without having to know the intricacies within R internals, or even R's C-facilities. However, this is unavoidable if one wishes to understand why the extendr-api is the way it is.
```

Comment:
I think this should probably get moved to some introductory page for the entire user guide.

Might be better to start each tutorial with an itemized list of learning objectives:

> In this tutorial, we explain how to work with vectors in R:
> * passing single values to Rust and back to R
> * passing multiple values to Rust and back to R
> * working with different data types (numeric, integer, character)